pkg/orchestration/wiring/registry/registry.go

```diff
@@ -30,5 +31,6 @@ var KnownWiringPlugins = map[voyager.ResourceType]wiringplugin.WiringPlugin{
 	rds.ResourceType:            wiringutil.StatusAdapter(rds.New().WireUp),
 	sqs.ResourceType:            wiringutil.StatusAdapter(sqs.WireUp),
 	asapkey.ResourceType:        wiringutil.StatusAdapter(asapkey.New().WireUp),
+	saml.ResourceType:           wiringutil.StatusAdapter(saml.New().WireUp),
```

Comment:
This could probably just be `wiringutil.StatusAdapter(saml.WireUp)` if you remove the struct